File: Old-Setl2/fibonacci.c
#include <stdlib.h>
#include <stdio.h>

unsigned fibonacci(unsigned n) 
{
	if (n == 0) {
		return 1;
	} else if (n == 1) {
		return 1;
	} else {
		return fibonacci(n-1) + fibonacci(n-2);
	}
}

int main() 
{
	unsigned i;
	for (i = 0; i < 50; ++i) {
		printf("fib(%d) = %d\n", i, fibonacci(i));
	}
	return 0;
}

	
	

File: Java/Fibonacci.java
public class Fibonacci 
{
    public static int fibonacci(int n) 
    {
        if (n == 0)
            return 1;
        if (n == 1)
            return 1;
        return fibonacci(n - 1) + fibonacci(n - 2);
    }

    public static void main(String[] args) 
    {
        for (int i = 0; i < 100; ++i) {
            int n = fibonacci(i);
            System.out.printf("fib(%d) = %d\n", i, n);
        }
    }    
}



